build/azure-pipelines/common/createAsset.ts

```diff
@@ -28,8 +28,10 @@ if (process.argv.length !== 8) {
 	process.exit(-1);
 }
 
+const Ignore = Symbol('Ignore');
```

Comment:
Why is this needed? It seems that if we intend to ignore something, we shouldn't call `createAsset` at all for it.